pkg/informer/event_handler.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2020 Red Hat, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package informer
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/client-go/tools/cache"
+)
+
+// EventHandlerMetrics stores the handled resources total metric pointer.
+type EventHandlerMetrics struct {
+	HandledTotal *prometheus.CounterVec
+}
+
+// NewEventHandlerMetrics takes in a prometheus registry and initializes and
+// registers the kube_events_exporter_handled_resources_total metric.
+// It returns this registered metric.
+func NewEventHandlerMetrics(registry *prometheus.Registry) *EventHandlerMetrics {
+	metrics := &EventHandlerMetrics{
+		HandledTotal: prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "kube_events_exporter_handled_resources_total",
```

Comment:
kube_events_exporter_handled_resources_total why do you think this metric is useful?

Reply:
I thought it might be insightful, but giving it another thought, I don't think this metric is useful.